doc/adr/0007-generate-qr-code-on-client

```diff
@@ -0,0 +1,25 @@
+# 1. Record architecture decisions
+
+Date: 2022-06-30
+
+## Status
+
+Accepted
+
+## Context
+
+QR Code is embedded in the eService page and it is used to allow user to start identification process via the UseID app camera which later. The QR Code contains `tcTokenURL`, `widgetSessionId` and `encryptionKey`. For this decision record, [ADR 0006](LINK TO 0006) 
+```
+eid://127.0.0.1:24727/eID-Client?tcTokenURL={{tcTokenURL}}&widgetSessionId={{widgetSessionId}}&mobile={{isMobileDevice}}#{{encryptionKey}}
```

Comment:
Not sure, if we should add the URL format here again? Makes it harder to change it, because more than 1 adr needs to be changed then. I would just remove it, since here it's not about the content of the QR code and only about where to create it. What's inside the QR code is already listed above.

Reply:
I think if we would like to change the scheme, we should create a new ADR entry in the future